ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -138,8 +138,13 @@ export class Send extends IronfishCommand {
           account: from,
           confirmations: flags.confirmations,
           assetId,
+          minimalBalance: 2n,
```

Comment:
This will prevent you from being able to send a transaction with a non-native asset if you only have 1n of that asset: 
<img width="877" alt="image" src="https://user-images.githubusercontent.com/57735705/222439926-27e3bd47-de39-43f9-a466-c898ec03ecc0.png">